python/lsst/ts/mthexapod/constants.py

```diff
@@ -68,31 +68,21 @@ class ControllerConstants:
         The command port is one larger than this value.
     sync_pattern : `int`
         Sync pattern for commands to the low-level controller.
-    config_frame_id : `int`
-        Frame ID for configuration messages from the low-level controller.
-    telemetry_frame_id : `int`
-        Frame ID for telemetry messages from the low-level controller.
     """
 
-    def __init__(self, port, sync_pattern, config_frame_id, telemetry_frame_id):
+    def __init__(self, port, sync_pattern):
         self.port = port
         self.sync_pattern = sync_pattern
-        self.config_frame_id = config_frame_id
-        self.telemetry_frame_id = telemetry_frame_id
 
 
 # Dict of SalIndex: ControllerConstants
 IndexControllerConstants = {
     enums.SalIndex.CAMERA_HEXAPOD: ControllerConstants(
         port=5560,
         sync_pattern=0x6666,
-        config_frame_id=0x1B,
-        telemetry_frame_id=0x7,
     ),
     enums.SalIndex.M2_HEXAPOD: ControllerConstants(
         port=5550,
         sync_pattern=0xB4B4,
-        config_frame_id=0x1C,
-        telemetry_frame_id=0x8,
     ),
 }
```

Comment:
Because the DDS telemetry will be the same as the GUI, the sync_pattern is 0xAAAA for telemetry data or 0xFFFF for the configuration data.
https://github.com/lsst-ts/ts_hexapod_controller/blob/develop/include/interface/telemetryStructure.h#L35-L39

Reply:
The sync patterns here are for commands sent to the low-level controller. I think the relevant code is here:
https://github.com/lsst-ts/ts_hexapod_controller/blob/4f5a8938a2a1190250b4575610f70ccbb5290bb4/src/interface/ddsCmdSocket.c#L162-L181
though it probably should be a pair of constants in a header file instead of buried in a source file.